content/webapp/components/SectionHeader/SectionHeader.tsx

```diff
@@ -34,17 +35,23 @@ const Title = styled.span`
 
 type Props = {
   title: string;
+  gridSize?: SizeMap;
 };
 
-const SectionHeader: FunctionComponent<Props> = ({ title }) => {
+const SectionHeader: FunctionComponent<Props> = ({ title, gridSize }) => {
   return (
     <div className={font('wb', 2)}>
-      <Layout gridSizes={gridSize12()}>
+      <ConditionalWrapper
+        condition={!!gridSize}
+        wrapper={children => (
+          <Layout gridSizes={gridSize12()}>{children}</Layout>
```

Comment:
should this be `<Layout gridSizes={gridSize}>{children}</Layout>` (since that's what's being passed in?)

Reply:
absolutely should, thanks for the catch

Reply:
Adressed [here](https://github.com/wellcomecollection/wellcomecollection.org/pull/11062/commits/44a53ba12f87165921ce9284609be1e18b0dc609)